.travis.yml

```diff
@@ -20,17 +20,19 @@ branches:
     - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
 
 before_install:
-  - curl -L "https://apt.llvm.org/llvm-snapshot.gpg.key" | sudo apt-key add -
-  - curl -L "http://packages.lunarg.com/lunarg-signing-key-pub.asc" | sudo apt-key add -
-  - echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main" | sudo tee -a ${TRAVIS_ROOT}/etc/apt/sources.list
-  - echo "deb https://packages.lunarg.com/vulkan bionic main" | sudo tee -a ${TRAVIS_ROOT}/etc/apt/sources.list
-  - sudo apt-get update
-  - sudo apt-get -yq --no-install-suggests --no-install-recommends install
-      llvm-${LLVM_VERSION}-dev
-      clang-${LLVM_VERSION}
-      clang-format-${LLVM_VERSION}
-      clang-tidy-${LLVM_VERSION}
-      spirv-tools
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      curl -L "https://apt.llvm.org/llvm-snapshot.gpg.key" | sudo apt-key add -
```

Comment:
Could the LLVM parts of this be guarded with a check for `$BUILD_EXTERNAL == 1`, and the `clang-format`/`clang-tidy` parts guarded with a check for `$CHECK_FORMAT == 1`? (We still need the SPIR-V tools, of course.)

Reply:
I'm not sure if we can guard LLVM parts as we might depend on tools that are not part of the build (because they take too long to build).  This feels out of scope of this change anyway.